tc-server/src/main/java/com/tc/l2/logging/TCLogbackLogging.java

```diff
@@ -122,6 +124,9 @@ public static void redirectLogging(File logDirFile) {
     } else {
       disableBufferingAppender(null);
     }
+    root.setLevel(Level.INFO);
```

Comment:
Is there any way to know if this has been set via config file?  I would hate to always force INFO if someone specifically configured DEBUG or WARN